### PR TITLE
DEV-5296 state profile add face value of loans

### PR DESCRIPTION
**Pull request by jameslee40**
**High level description:**
Adds face value of loans to State Profile Pages.

**Technical details:**

Technical details for the knowledge of other developers.

**JIRA Ticket:**
[DEV-1234](https://federal-spending-transparency.atlassian.net/browse/DEV-1234)

**Mockup:**
https://invis.io/RYA3XN5WP#/273832670_Homepage_2-2_E

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [N/A] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [N/A] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [x] Added Unit Tests for methods in Container Components, reducers, helper functions, and models `if applicable`
- [N/A] All `componentWillReceiveProps` and `componentWillMount` in relevant child/parent components/containers replaced with [future compatible life-cycle methods](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html) per [JIRA item 3277](https://federal-spending-transparency.atlassian.net/browse/DEV-3277)
- [x] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [N/A] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [N/A] Design review complete `if applicable`
- [x] [API #2759](https://github.com/fedspendingtransparency/usaspending-api/pull/2759) merged concurrently `if applicable`
- [x] Code review complete

### CODE REVIEW

File: src/js/components/award/shared/InfoTooltipContent.jsx [hunk 1]
```diff
@@ -1316,6 +1316,25 @@ export const recipientOverviewLoanInfo = (
     </div>
 );
 
+export const stateOverviewLoanInfo = (
```
Comment (jonhill13) [inline src/js/components/award/shared/InfoTooltipContent.jsx:1319]: This does not belong in this file considering these tooltips are for `award/shared/`. Probably, `components/state`.

File: src/js/components/state/overview/StateOverview.jsx [hunk 2]
```diff
@@ -109,15 +111,23 @@ export default class StateOverview extends React.PureComponent {
                         <strong>Note:</strong> All data on this page is based on Primary Place of Performance.
                     </div>
                     <div className="state-section__row">
-                        <div className="state-section__viz totals">
-                            <h3 className="state-overview__heading">
+                        <div className="state-section__viz totals-container">
```
Comment (jonhill13) [inline src/js/components/state/overview/StateOverview.jsx:114]: Not sure if you are seeing the same thing as me but i am seeing this locally.
Screen Shot 2020-09-03 at 12 30 48 PM
And this in prod.
Screen Shot 2020-09-03 at 12 39 39 PM
I guess design should weigh in on this.

**Review by jonhill13 — CHANGES_REQUESTED** [review]

File: src/js/components/award/shared/InfoTooltipContent.jsx (hunk 1 above)
Comment (jameslee40) [reply src/js/components/award/shared/InfoTooltipContent.jsx]: Moved it!

File: src/js/components/state/overview/StateOverview.jsx (hunk 2 above)
Comment (jameslee40) [reply src/js/components/state/overview/StateOverview.jsx:114]: Isn't this because the note was added above the award breakdown?

Comment (jameslee40) [reply src/js/components/state/overview/StateOverview.jsx:115]: @jonhill13 Conferred with design. We decided to move the note to the bottom below the table. Fixed.

**Review by jonhill13 — APPROVED** [review]

**Pull request merged**